src/mongocxx/test/spec/monitoring.hh

```diff
@@ -41,8 +41,26 @@ class apm_checker {
     void skip_kill_cursors();
     void print_all();
 
+    using event_vector = std::vector<bsoncxx::document::value>;
+    using iterator = event_vector::iterator;
+    using const_iterator = event_vector::const_iterator;
+
+    inline iterator begin() noexcept {
```

Comment:
Oh good idea. This does make checking capture events more convenient.

Reply:
Thanks!  It would have been difficult to implement these tests without this.